cpp/include/cudf/copying.hpp

```diff
@@ -14,14 +14,125 @@
  * limitations under the License.
  */
 
-#pragma once 
+#pragma once
 
+#include "cudf.h"
```

Comment:
This is redundant with `#include <cudf/cudf.h>`. Also, I think cudf.h should be avoided since it's largely just pulling in legacy headers.

Reply:
Typically you just need #include <cudf/types.hpp>.